src/SpanInterface.php

```diff
@@ -29,6 +29,13 @@ public function setAttribute(string $key, string|int|float|bool|Stringable $valu
      */
     public function setAttributes(iterable $attributes): SpanInterface;
 
+    /**
+     * Starts the current span.
+     *
+     * @return $this
+     */
+    public function start(): SpanInterface;
```

Comment:
this seemed to be missing (or it's not clear that `TracerInterface::createSpan` also starts a span? (in which case, I'll remove this and maybe update the description for `createSpan`...

Reply:
:+1: right! It was missing because `createSpan` was `startSpan` before #2, but now is needed.

Reply:
I believe the plan was `createSpan` no longer pre-starts the span

Reply:
Good spot. @alekitto is correct, we previously were pre-starting the span.
Since this is no longer the case, this is a welcome addition